lib/pure/testutils.nim

```diff
@@ -0,0 +1,15 @@
+## Utilities to help writing tests
+##
+## Unstable, experimental API.
+
+import std/[sequtils, algorithm]
+
+proc sortedPairs*[T](t: T): auto =
+  ## helps when writing tests involving tables in a way that's robust to
+  ## changes in hashing functions / table implementation.
+  toSeq(t.pairs).sorted
+
+template sortedItems*(t: untyped): untyped =
+  ## helps when writing tests involving tables in a way that's robust to
+  ## changes in hashing functions / table implementation.
+  sorted(toSeq(t))
```

Comment:
IMO you should use `toSeq(t.pairs).sorted` and `sorted(toSeq(t))` in the tests instead.
This is bloat IMO and saves like 5 keypresses.

Reply:
* sure, it's a bit "bloaty" but it simplifies tests (stdlib + other ones) and encourages using it instead of flaky alternatives like `$t` which has caused much churn in the past every time hashes/table/etc algorithms change; I've had to fix a number of tests (both in stdlib + in other nimble packages) to make those tests non-flaky; this simplifies this.
* it's a bit more than 5 keypresses in practice: `toSeq(t.pairs).sorted` prevents using method call syntax/UFCS (because pairs is an iterator), but `sortedPairs` can be used with MCS; in practice tests often involve expressions to define `t` so having MCS makes the test simpler to read/write
* (also, needs 1 import `testutils` instead of 2 `imports algorithm, sequtils` to achive this goal, but that argument is very minor; in practice `import sequtils` can cause further issues (see nim issues involving import sequtils except toSeq) so it'd in fact be `imports algorithm`, from sequtils import toSeq`; etc)

all those complications I've had in the past when fixing test led me to the simple realization that testutils is needed.

Reply:
As I said, use `toSeq(....` in the tests.
Even if it is only "a bit bloaty" it is too much to be a candidate for adding to the stdlib IMO.
All the other arguments are very minor, as you admit yourself.
Please remove this module.